testplan/common/utils/comparison.py

```diff
@@ -559,11 +562,12 @@ def _rec_compare(
     lhs,
     rhs,
     ignore,
-    only,
+    include,
     key,
     report_mode,
     value_cmp_func,
     _regex_adapter=RegexAdapter,
+    include_only_keys_of_rhs=False,
```

Comment:
include_per_rhs?

Reply:
I am going with include_only_rhs to use the same format as in the assertion